spec/models/manageiq/providers/redhat/infra_manager/refresh/refresher_graph_target_vm_spec.rb

```diff
@@ -57,6 +57,28 @@ def load_response_mock_for(filename)
       expect(saved_counted_models).to eq(counted_models)
     end
 
+    it 'refreshes vm hosts properly' do
+      allow(Spec::Support::OvirtSDK::ConnectionVCR).to receive(:new).with(kind_of(Hash)) do |opts|
+        Spec::Support::OvirtSDK::ConnectionVCR.new(opts,
+                                                   'spec/vcr_cassettes/manageiq/providers/redhat/infra_manager/refresh/ovirt_sdk_refresh_graph_target_vm_with_host.yml',
+                                                   false)
+      end
+      stub_settings_merge(:ems_refresh => { :rhevm => {:inventory_object_refresh => true }})
+      EmsRefresh.refresh(@ems)
+      @ems.reload
+      vm = VmOrTemplate.where(:name => "vm1").first
+      expect(vm.ems_id).to eq(@ems.id)
```

Comment:
we should probably check if the host relation is there? The check below might be just testing the host is not there in both cases.

Reply:
Looks like this is now checking that there is a host_id, I'd still like to add a check that it is actually the right host but that can be done in a followup